ansible/roles/stack-sunbird/templates/sl-assessments.env

```diff
@@ -0,0 +1,98 @@
+# Service Config
+# Port on which the service will listen to
+PORT={{sl_assessments_port | default("7201")}} 
+
+#Application ENV example local/dev/production
+NODE_ENV={{sl_assessments_node_env}}
+
+#Log configuration
+#Enable console logging
+ENABLE_DEBUG_LOGGING={{sl_assessments_enable_debug_logging | default("ON")}}
+
+#Enable writing logs to a file
+ENABLE_BUNYAN_LOGGING={{sl_assessments_enable_bunyan_logging | default("OFF")}}
+
+#service based URL
+APPLICATION_BASE_URL={{sl_assessments_application_base_url | default('/assessment/')}}
+
+# Allow report request to run for longer time
+request_timeout_for_reports={{sl_assessments_request_timeout_for_reports | default("600000")}}
+AUTHORIZATION={{sl_assessments_authorization}}
```

Comment:
What's the authorization? @aks30

Reply:
@rjshrjndrn - It will hold the bearer auth token value to access user/org apis, should we remove this? Since it's going to be in the same infra we will no longer need this right?

Reply:
as per our discussion i am removing AUTHORIZATION={{sl_assessments_authorization}}